traffic_portal/app/src/common/modules/table/_table.scss

```diff
@@ -19,6 +19,7 @@
     table {
         width: 100% !important;
     }
+    overflow: auto;
```

Comment:
Instead of this, you could set [the scrollx option](https://datatables.net/reference/option/scrollX) on the table. With this overflow, you can still leave pagination and other controls behind as you scroll, though the sidebar and title stay put properly. I tested with `"sScrollX": "100%";` and got the desired behavior of scrolling only on the table, but the docs claim that `scrollX: true` will work just as well.

Reply:
so many css properties....ok, thanks. i'll try it out

Reply:
That's not CSS, it's an option to the datatable initialization

Reply:
updated. good find.